indico/modules/events/papers/forms.py

```diff
@@ -86,8 +86,8 @@ class PaperReviewingSettingsForm(IndicoForm):
     RATING_FIELDS = ('scale_lower', 'scale_upper')
 
     announcement = IndicoMarkdownField(_('Announcement'), editor=True)
-    scale_lower = IntegerField(_("Scale (from)"), [UsedIf(lambda form, field: not form.has_ratings), InputRequired()])
-    scale_upper = IntegerField(_("Scale (to)"), [UsedIf(lambda form, field: not form.has_ratings), InputRequired()])
```

Comment:
We still show a "scale cannot be changed" message below - probably that should be updated as well like you did in abstracts?